strings/base_string.h

```diff
@@ -326,6 +326,38 @@ WINRT_EXPORT namespace winrt
         {
             return rend();
         }
+        
+        inline bool starts_with(const wchar_t value) const noexcept
+        {
+            return empty() ? false : data()[0] == value;
+        }
+
+        inline bool ends_with(const wchar_t value) const noexcept
+        {
+            return empty() ? false : data()[size() - 1] == value;
+        }
+
+#if _HAS_CXX20
+        inline bool starts_with(const std::wstring_view another) const noexcept
+        {
+            return operator std::wstring_view().starts_with(another);
+        }
+
+        inline bool starts_with(const wchar_t* const pointer) const noexcept
+        {
+            return operator std::wstring_view().starts_with(pointer);
+        }
+
+        inline bool ends_with(const std::wstring_view another) const noexcept
```

Comment:
For all of these methods, `inline` is not required and should be using trailing/right/west `const` for consistency.